src/OpenConext/Profile/Value/Consent.php

```diff
@@ -95,4 +113,12 @@ public function getConsentType()
     {
         return $this->consentType;
     }
+
+    /**
+     * @return null|AttributeSet
+     */
+    public function getAttributes()
+    {
+        return $this->attributes;
```

Comment:
The above suggestion wasn't suggested in the assumption that the released attributes are present at all times. That's what the setter is for.

I'd also like a different object, but I assumed the `givenFor()` approach was as discussed with you @DRvanR. My bad.

Nothing wrong with "released attributes" vs. "attributes", though? :smiley: